srcpkgs/jless/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'jless'
 pkgname=jless
-version=0.7.2
+version=0.8.0
 revision=1
 build_style=cargo
+makedepends="python3 xcb-util-devel"
```

Comment:
why python?

Reply:
the xcb crate is used here in an older version, and part of its build script is written in python

Reply:
should be a hostmakedepends then